### ISSUE TITLE
Use Symbol

### ISSUE THREAD

**Issue by ai** [polemius/undo#2]
User’ app can already use `undo` and `redo` events or `undoable` state. If we will use `Symbol` we will be more isolated.

```js
import { UNDO } from '@storeon/undo`

dispatch(UNDO)
```

**Comment by polemius** [comment]
I've crate the draft PR #4. Are you mean something like this? or are you mean use the:

```
var UNDO = Symbol('undo')
```

**Comment by ai** [comment]
Yeap

**Comment by polemius** [comment]
If I do this:

```
var UNDO = Symbol('undo')
dispatch(UNDO)
```

than I've got the error:

```
Unknown Storeon event Symbol(undo)
```

Can you please help me

**Comment by ai** [comment]
Did you changed `store.on(UNDO, …)`?

**Comment by polemius** [comment]
Oh, I'm sorry, I fogot changed also `store.on`.
I've updated my [PR](#4). Please check it out.

**Comment by ai** [comment]
Looks good, but maybe you should combine these changes with this idea? https://github.com/polemius/undo/issues/3

**Comment by polemius** [comment]
Ok, I've merged the #4 and will be working on the ideas from #3
